en-us/faq.md

```diff
@@ -0,0 +1,32 @@
+
+### So I just copy the mods, right?
+
+- No. Fabulously Optimized consists of mods, configs and resourcepacks. The settings would not be applied if you simply copied the mods over, and because you would have to setup the mods manually, you might not have the right settings to precisely match OptiFine.
+
+### What is the MultiMC ZIP?
+
+- The MultiMC ZIP is a packaged instance for the launcher called MultiMC. It is distributed in CurseForge because MultiMC itself no longer supports downloading CurseForge modpacks and downloads from CurseForge support the creator of FO. This also allows for an easy installation of FO for vanilla launcher users.
+
+### What is in the ZIP file?
+
+- In both the CurseForge and MultiMC ZIP files, they contain configs for each mod and resource packs which are essential for FO to run. The config folder contain configs for each mod in FO so the modpack is configured perfectly when you first start up the modpack, the mods folder contains mods that are essential to replicate OptiFine's features, and the resource folder contains a resource pack that helps the user navigate through Mod Menu. There is also a file called 'Copy all 3 folders!', this file makes it easier to update the pack rather than having to download and make a new instance each time there is a new update.
+
+### Why try a different launcher?
+
+- There are a variety of launchers available, each with unique features. If you use the vanilla launcher, you miss out on many features, including the ability to update the modpack and the version of Minecraft, add any mod with a few simple clicks, and finally download the modpack more easily.
+
+### Why is FO not a client?
+
+- FO is also not a client because simply we would need explicit permission from the mod authors to use their mods in a client and the authors of each mod would receive no benefits by being included in a client. We would also need to be able to code a client which would require lots of time and maintenance to keep the client up to date with the latest version of Minecraft. By being a modpack, this makes it easier for the creator and user to add their own mods in and use Fabulously Optimized as base pack for their own mods they want to add in. This can also make it easier to update the modpack by adding in replacement mods while waiting for other mods in the pack to update.
+
+### Is there an easier way to install FO?
+
+- Soon, there will be a vanilla installer that will make it easier to install the modpack on the vanilla launcher. You can check out the [Github Repo](https://github.com/Fabulously-Optimized/vanilla-installer/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc) for the installer to see when a beta could be released. We advise using other launchers like CurseForge Launcher, MultiMC, and GDLauncher to make it simple and easy to install and update the modpack for the time being until the vanilla installer is released.
```

Comment:
```suggestion
- Soon, there will be a program that will make it easier to install the modpack on the vanilla launcher. You can check out the [GitHub Repo](https://github.com/Fabulously-Optimized/vanilla-installer/releases) for the installer to see when a beta could be released. We advise using other launchers like CurseForge Launcher, MultiMC, and GDLauncher to make it simple and easy to install and update the modpack for the time being until the vanilla installer is released.
```
We probably won't announce in an issue when a beta of VI has been released, so link to the releases page. Otherwise just of couple spelling fixes :P